src/Shared/src/ReferenceAssemblyInfo.cs

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+// All reference assemblies should have the 0x70 flag which prevents them from loading
+// and the ReferenceAssemblyAttribute.
+[assembly:System.Runtime.CompilerServices.ReferenceAssembly]
+[assembly:System.Reflection.AssemblyFlags((System.Reflection.AssemblyNameFlags)0x70)]
```

Comment:
What flags are these and why can't we use the enum?

Reply:
This flag sets AssemblyName.ProcessorArchitecture to None. There is no public API for this. https://github.com/dotnet/coreclr/blob/64ca544ecf55490675e72b853e98ebc8cc75a4fe/src/System.Private.CoreLib/src/System/Reflection/AssemblyName.CoreCLR.cs#L74

Reply:
Add a comment and link to the source in our source?

Reply:
Done